apps/framework-cli/src/cli/commands.rs

```diff
@@ -188,6 +183,27 @@ pub enum GenerateCommand {
         #[arg(long, default_value = "false")]
         save: bool,
     },
+    /// Build a ClickHouse HTTPS connection string (prompts if args omitted)
+    ClickhouseUrl {
```

Comment:
i don't think this command is needed if we print out the connection string in the other command